sdk-commons/src/test/java/org/openmrs/maven/plugins/model/SdkStatisticsTest.java

```diff
@@ -4,15 +4,25 @@
 import org.junit.Test;
 
 import java.io.File;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Properties;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 /**
  *
  */
 public class SdkStatisticsTest {
 
+    private static final String DATE_FORMAT = "dd-M-yyyy";
```

Comment:
What is the use of this constant?

Reply:
to format the dates for `statsLastReported` property.